src/core/api/index.js

```diff
@@ -45,19 +59,29 @@ export function callApi({
   const apiURL = `${API_BASE}/${endpoint}/${queryString}`;
 
   return fetch(apiURL, options)
-    .then((response) => {
+    .then((response) => response.json().then(
```

Comment:
The second `then` should get dropped down a line. Moving it outside of the `.json()` call would probably be nicer.

```js
fetch()
  .then((response) => response.json())
  .then((jsonResponse) => ..., ...)
  .then(({ response, jsonResponse }) => ...);